src/widgets/menu/lv_menu.c

```diff
@@ -142,7 +142,12 @@ lv_obj_t * lv_menu_page_create(lv_obj_t * menu, char const * const title)
 
 lv_obj_t * lv_menu_cont_create(lv_obj_t * parent)
 {
-    LV_ASSERT_OBJ(parent, &lv_menu_page_class);
+    LV_ASSERT_NULL(parent);
+    if(!parent || !(parent->class_p == &lv_menu_page_class ||
+                    parent->class_p == &lv_menu_section_class)) {
```

Comment:
Maybe support descendant classes too, as `LV_ASSERT_OBJ` does this and, in theory, users are using descendant classes of menu_page in the wild right now.

```suggestion
    if(!parent || (LV_USE_ASSERT_OBJ &&
                   !(lv_obj_has_class(parent->class_p, &lv_menu_page_class)
                     || lv_obj_has_class(parent->class_p, &lv_menu_section_class)))) {
```

Reply:
Perfect suggestion with the usage of LV_USE_ASSERT_OBJ 💯